.circleci/config.yml

```diff
@@ -77,10 +77,16 @@ jobs:
               docker-compose ps
               sleep 45
 
+              docker-compose exec harness-cli bash -c 'sed -i "s|training_sleep_seconds=40|training_sleep_seconds=60|g" /harness-cli/examples/ur/simple-integration-test.sh'
```

Comment:
@DioniPinho on my laptop 60 is not enough, I use 90. And resources that we use for ci for now is smaller then my laptop. I'd suggest something like 240 or 300